twins/README.md

```diff
@@ -26,11 +26,11 @@ default values.
 | MF_TWINS_SINGLE_USER_TOKEN | User token for single user mode that should be passed in auth header |                       |
 | MF_TWINS_CLIENT_TLS        | Flag that indicates if TLS should be turned on                       | false                 |
 | MF_TWINS_CA_CERTS          | Path to trusted CAs in PEM format                                    |                       |
-| MF_TWINS_MQTT_URL          | Mqtt broker URL for twin CRUD and states update notifications        | tcp://localhost:1883  |
-| MF_TWINS_CHANNEL_ID        | Mqtt notifications topic                                             |                       |
+| MF_TWINS_CHANNEL_ID        | NATS notifications channel id                                        |                       |
+| MF_TWINS_SUBTOPIC_WILDCARD | Subtopic wildcard for attribute's definition                         | #                     |
```

Comment:
Why this is configurable? Are you using MQTT to get these attributes? If yes, then "catch all joker can not be anything else (it can be `+` for a singe-level topic, though).